shinhye/app.js

```diff
@@ -47,7 +48,135 @@ app.post("/users/signup", async (req, res) => {
     [username, firstName, lastName, age, email, password]
   );
 
-  res.status(201).json({ message: "Successfully created!" });
+  return res.status(201).json({ message: "Successfully created!" });
+});
+
+// add a post
+app.post("/posts", async (req, res) => {
+  const { userID, userProfileImage, postImageURL, postContent } = req.body;
+
+  await AppDataSource.query(
+    `INSERT INTO posts(
+      user_id, 
+      user_profile_image,
+      post_image_url,
+      post_content
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [userID, userProfileImage, postImageURL, postContent]
+  );
+
+  return res.status(201).json({ message: "Successfully created!" });
+});
+
+// view all posts
+app.get("/posts", async (req, res) => {
+  const postData = await AppDataSource.query(
+    `
+    SELECT
+    user_id as userID,
+    user_profile_image as userProfileImage,
+    id as postId,
+    post_image_url as postImageUrl,
+    post_content as postContent
+    FROM posts;
+    `
+  );
+
+  return res.status(200).send({ data: postData });
+});
+
+// view user's post
+app.get("/posts/:userID", async (req, res) => {
+  const { userID } = req.params;
```

Comment:
마이너하지만 path-parameter 네이밍 수정해주세요!
관련된 query 문도 수정해주세요.
```suggestion
  const { userId } = req.params;
```

Reply:
넵 반영했습니다!